config/fakro/arz-z-wave.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+  <!-- Configuration -->
+  <CommandClass id="112">
+  </CommandClass>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
```

Comment:
But from where i can get Association Group ? Becuase now when i use it in domoticz it has no association group.
![ass](https://cloud.githubusercontent.com/assets/16692223/12710352/134fe592-c8b6-11e5-9d92-2c1d3de2f6d3.jpg)